_data/events.yml

```diff
@@ -27,3 +27,8 @@
   city: Edmonton
   province: AB
   url: https://www.epl.ca/odd/
+  
+- name: 'Ottawa Open Data Day 2018'
+  city: Ottawa
+  province: ON
```

Comment:
the yaml spec has an inconvenient property when "on" without quotes is registered as a boolean ;)

To speed things up, I'll make the fix.